scripts/run-jenkins.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash -xe
+
+TIMEOUTCMD=scripts/babysitter
```

Comment:
I think we did it because we weren't sure if it works well. I agree we can just enable it now.